IanGRap/Squish
# Audio updates 5 31

Contributors: Fernando (aka Sound God), <sub>Alfredo</sub>

Goal: Try to make this the final Audio Update. Fernando did slight remixing/ reorganizing of the files. 

Files Changed: 
* CameraManager.cs: Removed a call to an old Block Break sound
* Moved a bunch of audio files into more organized sub folders
* GT_(song) : Changed the compression settings for the game play songs
* AudioManager.prefab: Added new songs into the game play music lists. 

How to Test: 
* Run through the whole game. 
* Make sure that the audio still sounds great. Because some audio files were deleted, make sure that there are no warnings in the console that show that one of the deleted files is trying to be called. 
* Test the audio in a Build as well. (This is important, we want to make sure the audio sounds aren't different in a build)

* Note: I accidentally left the 5 31 in the title of the pull request. This is just the date but it took out the dashes (-).
